src/applications/vaos/appointment-list/components/PastAppointmentsListV2/index.jsx

```diff
@@ -0,0 +1,264 @@
+import {
+  FETCH_STATUS,
+  GA_PREFIX,
+  APPOINTMENT_TYPES,
+} from '../../../utils/constants';
+import { getVAAppointmentLocationId } from '../../../services/appointment';
+import AppointmentListItem from '../AppointmentsPage/AppointmentListItem';
+import ExpressCareListItem from '../AppointmentsPage/ExpressCareListItem';
+import NoAppointments from '../NoAppointments';
+import moment from 'moment';
+import PastAppointmentsDateDropdown from '../PastAppointmentsList/PastAppointmentsDateDropdown';
+
+export function getPastAppointmentDateRangeOptions(today = moment()) {
+  const startOfToday = today.clone().startOf('day');
+
+  // Past 3 months
+  const options = [
+    {
+      value: 0,
+      label: 'Past 3 months',
+      startDate: startOfToday
+        .clone()
+        .subtract(3, 'months')
+        .format(),
+      endDate: today.format(),
+    },
+  ];
+
+  // 3 month ranges going back ~1 year
+  let index = 1;
+  let monthsToSubtract = 3;
+
+  while (index < 4) {
+    const start = startOfToday
+      .clone()
+      .subtract(index === 1 ? 5 : monthsToSubtract + 2, 'months')
+      .startOf('month');
+    const end = startOfToday
+      .clone()
+      .subtract(index === 1 ? 3 : monthsToSubtract, 'months')
+      .endOf('month');
+
+    options.push({
+      value: index,
+      label: `${start.format('MMM YYYY')} – ${end.format('MMM YYYY')}`,
+      startDate: start.format(),
+      endDate: end.format(),
+    });
+
+    monthsToSubtract += 3;
+    index += 1;
+  }
+
+  // All of current year
+  options.push({
+    value: 4,
+    label: `All of ${startOfToday.format('YYYY')}`,
+    startDate: startOfToday
+      .clone()
+      .startOf('year')
+      .format(),
+    endDate: startOfToday.format(),
+  });
+
+  // All of last year
+  const lastYear = startOfToday.clone().subtract(1, 'years');
+
+  options.push({
+    value: 5,
+    label: `All of ${lastYear.format('YYYY')}`,
+    startDate: lastYear.startOf('year').format(),
+    endDate: lastYear
+      .clone()
+      .endOf('year')
+      .format(),
+  });
+
+  return options;
+}
+
+function PastAppointmentsListNew({
+  showScheduleButton,
+  dateRangeOptions = getPastAppointmentDateRangeOptions(),
+  appointmentsByMonth,
+  pastStatus,
+  facilityData,
+  fetchPastAppointments,
+  startNewAppointmentFlow,
+  pastSelectedIndex,
+  showPastAppointments,
+}) {
+  const history = useHistory();
+  const [isInitialMount, setInitialMount] = useState(true);
+  useEffect(() => {
+    if (!showPastAppointments) {
+      history.push('/');
+    } else if (pastStatus === FETCH_STATUS.notStarted) {
+      const selectedDateRange = dateRangeOptions[pastSelectedIndex];
+      fetchPastAppointments(
+        selectedDateRange.startDate,
+        selectedDateRange.endDate,
+        pastSelectedIndex,
+      );
+    }
+  }, []);
+  useEffect(
+    () => {
+      if (pastStatus === FETCH_STATUS.succeeded && !isInitialMount) {
+        focusElement('#queryResultLabel');
```

Comment:
This element doesn't exist, but we do need to set focus somewhere after the date range is changed. Maybe the first month header (the h3s down below)?

Reply:
Makes sense, focusing on h3 below